test/integration/fixtures/publishing_api_payload.json

```diff
@@ -0,0 +1,216 @@
+{
+	"title": "Pitts S-2B, G-SKYD, 21 June 1996",
```

Comment:
any reason we have used 4 space indentation? the existing JSON file is using 2 space indentation.